Signal/src/Profiles/OWSProfileManager.m

```diff
@@ -82,31 +82,24 @@ - (NSUInteger)hash
 
 #pragma mark -
 
+NSString *const kNSNotificationName_LocalProfileUniqueId = @"kNSNotificationName_LocalProfileUniqueId";
```

Comment:
You mean that prefix?  I've used that (many places) because they are posted with the method `postNotificationName:`.  If you want we can just use the prefix `kNSNotification_`, NBD.